### PR TITLE
Multi-Z look up verb

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[content]

## What this does
adds this, was requested a long time ago, requires a clear visible space above your own turf to work

## Why it's good
nice to be able to see above too

## Changelog
:cl:
 * rscadd: Players can now look up one connected Z level on multi-Z maps.
 * bugfix: The move up and down verbs no longer show if there is no connected Z level to move to for each.

### CODE REVIEW

**Review by jwhitak — APPROVED** [review]
Would there be a way to hide this verb on z-levels/maps without multi-z? Not a deal breaker but just a thought.

**Comment by SECBATON-GRIFFON** [comment]
> Would there be a way to hide this verb on z-levels/maps without multi-z? Not a deal breaker but just a thought.

move up and down currently isn't hidden on them either, i'll look into it

edit: done

File: code/modules/multiz/movement.dm [hunk 1]
```diff
@@ -12,6 +12,28 @@
...
+		if(client.eye == T)
+			lookup_reset_view()
+		else if(T && isvisiblespace(T))
+			reset_view(GetAbove(src))
+			to_chat(src, "<span class='notice'>You look upwards.</span>")
+			register_event(/event/moved, src, nameof(src::lookup_reset_view()))
+		else
+			to_chat(src, "<span class='warning'>You see nothing but the ceiling.</span>")
+	else
+		to_chat(src, "<span class='warning'>You don't feel like doing that.</span>")
```
Comment (west3436) [inline code/modules/multiz/movement.dm:30]: I'd rather this verb just be disabled on non-multiz maps if possible

Comment (SECBATON-GRIFFON) [reply code/modules/multiz/movement.dm:30]: done

**Pull request merged**